openlibrary/i18n/__init__.py

```diff
@@ -91,11 +91,19 @@ def compile_translations():
         if os.path.exists(po_path):
             _compile_translation(po_path, mo_path)
 
-def update_translations():
+
+def update_translations(locales=[]):
```

Comment:
There's an ugly little gotcha with setting lists as the default. If I recall correctly, python keeps the list around, and doesn't give you a new list each time the function is called :/ So if you append something to that list, next time this function is called, the default will be different!

```suggestion
def update_translations(locales):
```

Reply:
I think that either you or Chris told me about this some time ago.  Thanks for the reminder!